e2e/addons/addons_test.go

```diff
@@ -33,6 +33,7 @@ var _ = Describe("addons flow", func() {
 
 		It("app init creates an copilot directory", func() {
```

Comment:
maybe "app init creates an copilot directory and a workspace file"? since you validate for the file as well.